curvefs/conf/client.conf

```diff
@@ -204,6 +204,18 @@ s3.writeCacheMaxByte=838860800
 s3.readCacheMaxByte=209715200
 # file cache read thread num
 s3.readCacheThreads=5
+
+# The data in the cache cluster download to local
+s3.memClusterToLocal=true
```

Comment:
The s3.memClusterToLocal and s3.s3ToLocal can be merged into one options (maybe s3.S3ToLocalFirst)?

Reply:
these are two separate modules and functions. so ...

Reply:
It is a little confused that s3.memClusterToLocal And s3.s3ToLocal are both true or false?

Reply:
> It is a little confused that s3.memClusterToLocal And s3.s3ToLocal are both true or false?

they are independent, as you see follows：
` 
  

    if (!kvClientManager_ && FLAGS_s3ToLocal) {   // It is configurable whether to write to local cache or not
        // get from s3 directly
        PrefetchS3Objs(prefetchObjs, true);
    } else if (FLAGS_memClusterToLocal) {
        // get from memcached first, if failed, get from s3
        PrefetchS3Objs(prefetchObjs, false);
    }
`

- if memcached cluster is exist:
  - FLAGS_memClusterToLocal=true: download objs from memcached
  - FLAGS_memClusterToLocal=false: download is disabled
-  if memcached cluster is not exist:
  -  FLAGS_s3ToLocal=true: download objs from s3
  - FLAGS_s3ToLocal=false: download is disabled

Reply:
ok